e2e-tests/src/test/java/uk/nhs/adaptors/gp2gp/e2e/EhrExtractTest.java

```diff
@@ -73,6 +74,14 @@ public void When_ExtractRequestReceivedForPatientWithNoDocs_Expect_DatabaseToBeU
         var gpcAccessDocument = waitFor(() -> emptyDocumentTaskIsCreated(conversationId));
         assertThatNotDocumentsWereAdded(gpcAccessDocument);
 
+        var ackToRequester = (Document) waitFor(() -> Mongo.findEhrExtractStatus(conversationId).get("ackToRequester"));
+        assertThatAcknoledgementToRequestWasSent(ackToRequester);
+    }
+
+    private void assertThatAcknoledgementToRequestWasSent(Document ackToRequester) {
+        assertThat(ackToRequester.get("messageId")).isNotNull();
+        assertThat(ackToRequester.get("taskId")).isNotNull();
+        assertThat(ackToRequester.get("typeCode")).isEqualTo(ACCEPTED_ACKNOWLEDGEMENT_TYPE_CODE);
```

Comment:
Typo in method name, and we could use `assertAll(...)` to get as much info as possible on failure

```suggestion
    private void assertThatAcknowledgementToRequestWasSent(Document ackToRequester) {
        assertAll(
            () -> assertThat(ackToRequester.get("messageId")).isNotNull(),
            () -> assertThat(ackToRequester.get("taskId")).isNotNull(),
            () -> assertThat(ackToRequester.get("typeCode"))
                .isEqualTo(ACCEPTED_ACKNOWLEDGEMENT_TYPE_CODE)
        );
```